chain/beacon/node_test.go

```diff
@@ -546,10 +548,10 @@ func TestBeaconThreshold(t *testing.T) {
 	verifier := chain.NewVerifier(sch)
 
 	currentRound := uint64(0)
-	var counter = &sync.WaitGroup{}
+	var counter sync.WaitGroup
```

Comment:
It's preferrable to create non-pointer values if they are not meant to be shared. When the time comes to share them with other goroutines/functions, then the user can decide over using a value vs a pointer. That allows removing a whole class of bugs caused by mistakes in sharing memory rather than values.

Reply:
@ardan-bkennedy anything else to add for this change?